server/src/main/resources/application.properties

```diff
@@ -1,2 +1,5 @@
 quarkus.http.test-port=8083
-io.delta.sharing.api.server.defaultMaxResults=10
\ No newline at end of file
+io.delta.sharing.api.server.defaultMaxResults=10
+
+# aws
+provider.aws.region=eu-west-1
```

Comment:
this should be picked up from the storage object, not from application configuration.